tests/test_message_fields.py

```diff
@@ -6,19 +6,33 @@
 class TestMoneyField():
 
     def test_init(self):
-        amount = 1
+        amount = 100
         currency = 'SEK'
+        vat_amount = 25
+        vat_rate = 0.25
 
-        mf = MoneyField(amount=amount, currency=currency)
+        mf = MoneyField(
+            amount=amount,
+            currency=currency,
+            vat_amount=vat_amount,
+            vat_rate=vat_rate
+        )
 
         assert mf.amount == amount
         assert mf.currency == currency
 
     def test_to_decimals_should_return_decimal(self):
         amount = 1000
         currency = 'SEK'
-
-        mf = MoneyField(amount=amount, currency=currency)
+        vat_amount = 25
```

Comment:
Not to be more annoying than necessary, but vat amount should be 250 in this case if vat rate is 25%. :+1:

Reply:
👍